pages/base-page.js

```diff
@@ -145,6 +145,7 @@ exports.BasePage = class BasePage {
 
   async waitInfoMessageHidden() {
     await this.infoMessage.waitFor({ state: 'hidden' });
+    await this.page.waitForTimeout(400);
```

Comment:
Couldn't we replace the [timeout](https://playwright.dev/docs/api/class-page#page-wait-for-timeout) to any other [alternatives](https://stackoverflow.com/questions/46919013/puppeteer-wait-n-seconds-before-continuing-to-the-next-line/73676564#73676564)? 

Just as a tip, could we perhaps to wait for a request to be performed? As I see it in the tests that use this method it's being called to 
https://design.penpot.dev/api/rpc/command/update-profile-photo
https://design.penpot.dev/api/rpc/command/update-team-photo

Reply:
The issue is that the API request returns 200 before the image actually changes… So, while everything on the UI indicates that the photo has changed, the animation hasn’t yet completed. I’ll try a few more approaches.

If they don’t work, I might stabilize the test with a condition to 'retry the snapshot until it’s successful,' with a maximum of 4 iterations. This way, we can avoid hardcoding a wait time.

Reply:
Thanks for the explanations. It all sounds totally understandable.

Reply:
@chalapkoStanislav I understand you're still working on those approaches. Am I right?

Reply:
no, I already requested a recheck =)
please look at the last commit